clang/include/clang/Driver/Options.td

```diff
@@ -3007,6 +3003,11 @@ defm clangir_direct_lowering : BoolFOption<"clangir-direct-lowering",
   FrontendOpts<"ClangIRDirectLowering">, DefaultTrue,
   PosFlag<SetTrue, [], [ClangOption, CC1Option], "Lower directly from ClangIR to LLVM">,
   NegFlag<SetFalse, [], [ClangOption, CC1Option],  "Lower through MLIR to LLVM">>;
+defm clangir_call_conv_lowering : BoolFOption<"clangir-call-conv-lowering",
```

Comment:
Please add a comment here saying this is temporary and shall be removed once all existing tests migrate to use callingconv code.